src/Server/src/Core/SecretService.cs

```diff
@@ -43,12 +45,24 @@ public async Task<(Secret, string)> CreateSecretAsync(AddSecretRequest request)
 
             if (request.SaveValue.GetValueOrDefault())
             {
-                secret.EncryptedSecret = await _encryptionService.EncryptAsync(secretValue, CancellationToken.None);
+                secret.EncryptedSecret =
+                    await _encryptionService.EncryptAsync(secretValue, CancellationToken.None);
                 secret.EncryptionKeyId = await _encryptionService.GetEncryptionKeyNameBase64Async();
             }
 
             return (secret, secretValue);
         }
 
+        public async Task<string> GetDecryptedSecretAsync(Secret secret,
```

Comment:
We usually format it like this:
```suggestion
        public async Task<string> GetDecryptedSecretAsync(
            Secret secret,
```

either all or no parameter wrapped